File: app1/src/main/java/com/zinc/flavordemo/MainActivity.java
package com.zinc.flavordemo;

import android.os.Bundle;
import android.widget.TextView;

import androidx.appcompat.app.AppCompatActivity;
/*
https://blog.csdn.net/weixin_37625173/article/details/102510549
 */
public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        TextView tvContent = findViewById(R.id.tv_content);
        tvContent.setText("APPLICATION_ID:" + BuildConfig.APPLICATION_ID + "\n" +
                "VERSION_NAME:" + BuildConfig.VERSION_NAME + "\n" +
                "VERSION_CODE:" + BuildConfig.VERSION_CODE + "\n" +
                "LogicUtils:" + LogicUtils.calculate(1, 2));


    }
}

File: app/src/main/java/com/example/project1/MainActivity.kt
package com.example.project1

import android.os.Bundle
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import com.example.applib2.Lib2

class MainActivity : AppCompatActivity() {
    private var textView:TextView?=null
    private var textView2:TextView?=null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        textView=findViewById(R.id.test)
        textView2=findViewById(R.id.test2)
        textView2?.text= this.packageName
        textView?.text=Lib2().testLib2()

    }
    fun test(){

    }

    
}
